src/components/footer-indicator/FooterIndicator.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../styles/variables';
+
+.bdl-FooterIndicator {
+    background: $fours;
+    border-radius: 4px 4px 0 0;
+    bottom: 0;
+    height: 32px;
+    margin: 0 15px;
+    position: fixed;
+    width: 170px;
+    z-index: 10;
+}
+
+.bdl-FooterIndicator-content {
+    display: flex;
+    justify-content: flex-start;
+
+    &:focus {
+        outline: none;
+    }
+}
+
+.bdl-FooterIndicator-iconWrapper {
+    margin: 8px 9px 8px 10px;
+}
+
+.bdl-FooterIndicator-text {
+    color: $white;
+    font-size: 11px;
+    margin: 6px 10px 6px 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+
+    &::after {
```

Comment:
Is this needed? If so, a comment may be helpful.

Reply:
I originally added this as a workaround to hide tooltips for truncated content in Safari (https://stackoverflow.com/questions/20974276/prevent-safari-from-showing-tooltip-when-text-overflow-is-hidden-with-ellipsis). See LeftSidebarLink.scss for an example. However, I have just confirmed that we can show the tooltip regardless of the length of the text, so I will remove this rule 🙂